duckduckgo/macos-browser
# chore(deps): update Duck Player to use shared messaging

Task/Issue URL: https://app.asana.com/0/0/1203201846416004/f

**Description**:

The encryption mechanism for secure messaging in Catalina (used in Autofill and Duck Player) was consolidated into a single shared module - thanks @ayoy !

This PR follows that pattern, using a shared solution for the JS side of things.

**Steps to test this PR**:
1. Ensure Duck Player continues to work on Catalina
1. Ensure Duck Player continues to work on non-Catalina targets

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
